lib/gateways/fixed_data_gateway.rb

```diff
@@ -0,0 +1,9 @@
+class FixedDataGateway
+  def save(fixed_data)
+    FixedDatum.create(fixed_data)
+  end
+
+  def all
+    FixedDatum.all
```

Comment:
When using Rails' ActiveRecord from a gateway, I generally like to return a non-ActiveRecord object, as it prevents the rest of the system from being polluted by Rails. As this stands, the use case will receive an ActiveRecord object. This risks other parts of the system calling Rails model methods, breaking the gateway/use-case boundary.

I'd recommend creating a domain object called Domain::FixedDatum or similar, and mapping the model objects returned by FixedDatum.all into those domain objects. 

Shortened example of this from my WIP:

```
module Domain
  class Rent
    attr_accessor :uprn,
      :address,
      :address_2,
      ...

    def initialize(**config)
      config.each do |key, value|
        self.send("#{key}=", value)
      end
    end

    # example of how we can add dynamic attributes
    def lower_of_formula_rent_or_cap
      [formula_rent_this_year, rent_cap_this_year].min
    end
  end
end
```

Reply:
I haven't created a domain object for the fixed data because in this PR only saves the fixed data, rather than returning it (therefore I don't actually need the `all` method in`FixedDataGateway`. I've created a domain object for Rent Charge though